integration_tests/pages/link/requestLink.ts

```diff
@@ -40,4 +40,9 @@ export default class RequestLinkPage extends Page {
     cy.get('.govuk-error-summary__list').should('contain', partialMessage)
     cy.get('#email-error').should('contain', partialMessage)
   }
+
+  static goToPage = (): RequestLinkPage => {
```

Comment:
We tend to prefer this type of abstraction. It means that where possible we dont use `cy.` type commands in the spec files themselves, and we put all this implementation detail in the Page classes themselves. Its a bit cleaner, and means that in theory if we were to ever replace cypress with some other page driver technology the spec files themselves should not have to change